published/011-2018-sep-26.md

```diff
@@ -0,0 +1,62 @@
+
+## Updates and new releases
+*See the latest releases of IPFS tools and projects across the ecosystem.*
+
++ [go-ipfs 0.4.17](https://github.com/ipfs/go-ipfs/releases/tag/v0.4.17) – IPFS implementation in Go
++ [ipfs-companion v2.5.0.11550 (Beta)](https://github.com/ipfs-shipyard/ipfs-companion/releases/tag/v2.5.0.11550) – Browser extension that simplifies access to IPFS resources
++ [js-ipfs v0.32.0](https://github.com/ipfs/js-ipfs/releases/tag/v0.32.0) – IPFS implementation in JavaScript
++ [npm-on-ifps v0.6.4](https://github.com/ipfs-shipyard/npm-on-ipfs/releases/tag/v0.6.4) – 📦 Install npm modules through IPFS!
++ [Replicated Object Notation 2.1](https://github.com/gritzko/ron/blob/master/docs/Objectives%20for%20RON%2021.md) – Replicated Object Notation, a distributed live data format, golang/ragel lib
++ DNSLink Detection in IPFS Companion: [Our browser extension](https://github.com/ipfs/ipfs-companion) detects [DNSLink](http://docs.ipfs.io/guides/concepts/dnslink/) by default now. Visit [docs.ipfs.io](http://docs.ipfs.io) or [tr.wikipedia-on-ipfs.org](http://tr.wikipedia-on-ipfs.org) and you will load it from  IPFS, not HTTP! For more details: [DNSLink Support in IPFS Companion](https://github.com/ipfs-shipyard/ipfs-companion/blob/master/docs/dnslink.md)
++ [Qri 0.5.2](https://github.com/qri-io/qri/releases/tag/v0.5.2), or Prune Norwegian Buhund, is here!
++ [The Graph Development Update](https://medium.com/graphprotocol/the-graph-development-update-d90321e22748), 13 Sept 2018
++ [Polkadot PoC-2 is here](https://medium.com/polkadot-network/polkadot-poc-2-is-here-parachains-runtime-upgrades-and-libp2p-networking-7035bb141c25): Parachains, runtime upgrades, and Libp2p networkin
+
+## Tools and projects we <3
+*[Awesome IPFS](https://awesome.ipfs.io/) is a community maintained and updated list of projects, tools, or pretty much any things related to IPFS that are totally awesome. To see more, or add yours to the list, visit [Awesome IPFS on GitHub](https://github.com/ipfs/awesome-ipfs).* 
+
++ [Neocities hit 200K sites this month!](https://twitter.com/neocities/status/1039204355763666945)
++ [are we distributed yet?](https://arewedistributedyet.com/)
++ [Pinata](https://www.pinata.cloud/), deploy to IPFS effortlessly.
+
+## Coming up in the Community
+*Did you know IPFS has a community forum at [discuss.ipfs.io](https://discuss.ipfs.io/)? Sign up to participate in discussions about coding, tutorials, see announcements and learn about upcoming community events.*
+
++ [35th Chaos Communication Congress](https://events.ccc.de/2018/09/11/35c3-call-for-participation-and-submission-guidelines/) CFP is now open.
++ New Meetup Alert!: [peer to peer ÖRESUND](http://p2p-oresund.org/), in Malmö, Swededn, on October 27. 
++ [Pre-register for IPFS Conf in Lisbon.](https://docs.google.com/forms/d/e/1FAIpQLSfJVVPwvp6RY3MUg1zAVl1g_5y2nGb7WJIMI1Hs6glzm7FLHQ/viewform)
```

Comment:
cc @diasdavid @mikeal Is this still valid?

Reply:
It's still valid but it's not "pre-register" so much as "signup to be notified."

Reply:
@whyrusleeping note that the date changed, it says 2019 now